servicecontrol/installation.md

```diff
@@ -23,24 +23,32 @@ NOTE: ServiceControl should be installed on a separate dedicated machine with de
 
 ## Transport support
 
-In ServiceControl version 1.7 and above, the transport DLLs are managed by the installation and do not need to be downloaded from NuGet. ServiceControl can be configured to use one of the supported [transports](/transports/) listed below using the ServiceControl Management application:
+In ServiceControl version 1.7 and above, the transport packages are managed by the installation and do not need to be downloaded from NuGet. ServiceControl can be configured to use one of the supported [transports](/transports/) listed below using the ServiceControl Management application:
 
- * [Microsoft Message Queuing (MSMQ)](/transports/msmq/)
- * [Azure Storage Queues](/transports/azure-storage-queues/)
- * [Azure Service Bus](/transports/azure-service-bus/)
- * [Azure Service Bus .NET Standard](/transports/azure-service-bus-netstandard)
- * [SQL Server](/transports/sql/)
- * [RabbitMQ](/transports/rabbitmq/)
- * [Amazon SQS](/transports/sqs/)
+* [Microsoft Message Queuing (MSMQ)](/transports/msmq/)
+* [Azure Service Bus](/transports/azure-service-bus-netstandard)
+* [Azure Storage Queues](/transports/azure-storage-queues/)
+* [SQL Server](/transports/sql/)
+* [RabbitMQ](/transports/rabbitmq/)
+* [Amazon Simple Queue Service (SQS)](/transports/sqs/)
 
- Certain features of the transports are not supported natively by ServiceControl and will require a [transport adapter](/servicecontrol/transport-adapter).
- 
-For preliminary Azure Service Bus .NET Standard support, select the Azure Service Bus forwarding topology.
+### Legacy options
```

Comment:
I feel like this section needs an intro sentence or two that explains what this list is and why they are legacy. It should also talk about why we're mentioning migration options.

Reply:
I went with:

>The following options are considered legacy and migration to a non-legacy option should be considered before used.

Thoughts?